exareme2/controller/quart/app.py

```diff
@@ -3,15 +3,14 @@
 from exareme2.controller.quart.endpoints import algorithms
 from exareme2.controller.quart.error_handlers import error_handlers
 
-#
-# QUART_HOST = "0.0.0.0"
-# QUART_PORT = 5000
-# QUART_DEBUG = True  # Change to enable debug
-#
-
 app = Quart(__name__)
 app.register_blueprint(algorithms)
 app.register_blueprint(error_handlers)
 
+# Uncomment these lines only if you want to debug locally works only for the algorithm_validation_tests.
```

Comment:
`Uncomment if you want to debug locally. The app.run cannot receive host and port from here, since it will ignore the env variables passed from the dockerfile run command.`